.github/workflows/pre-merge.yaml

```diff
@@ -1,9 +1,16 @@
 name: Pre Merge Checks
-on: [push, pull_request]
+on: 
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '*'
```

Comment:
nit. If I remember correctly you might just ignore the `- '*'` line ;)

Reply:
:( 
![Screenshot 2020-05-15 at 18 24 38](https://user-images.githubusercontent.com/3001957/82073333-62ed2600-96d9-11ea-9f70-93d457e1d783.png)